tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -0,0 +1,55 @@
+package e2e
+
+import (
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/resource"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+// TestE2ECreatingAndDestroyingWindowsInstanceOnEC2 creates and terminates Windows instance on AWS.
+// TODO: After creation, check for the following properties on the instance:
+// - instance exist
+// - in running state
+// - has public subnet
+// - Public IP
+// - OpenShift cluster Worker SG
+// - Windows SG
+// - OpenShift cluster IAM
+// - has name
+// - has OpenShift cluster vpc
+// - instance and SG IDs are recorded to the json file
+// After termination, it checks for the following:
+// - instance terminated
+// - Windows SG deleted
+// - json file deleted
+func TestE2ECreatingAndDestroyingWindowsInstanceOnEC2(t *testing.T) {
+	// Get kubeconfig, AWS credentials, and artifact dir from environment variable set by the OpenShift CI operator.
+	kubeconfig := os.Getenv("KUBECONFIG")
+	awscredentials := os.Getenv("AWS_SHARED_CREDENTIALS_FILE")
+	dir := os.Getenv("ARTIFACT_DIR")
+
+	// The e2e test uses Microsoft Windows Server 2019 Base image, m4.large instance type, and libra ssh key.
+	// The CI-operator uses AWS region `us-east-1` which has the corresponding image ID: ami-0dbbd6f952e12feba for
```

Comment:
Does AMI correspond to `Windows Server 2019 with Container runtime`? If not please set the AMI ID to that.

Reply:
Your AMI ID in the code and comment do not match. Please fix and say `Microsoft Windows Server 2019 Base with Containers` instead of just `Windows`.